packages/mobile/src/screens/app-screen/AppTabNavigationProvider.tsx

```diff
@@ -27,14 +34,21 @@ export const AppTabNavigationProvider = (
   const { children } = props
   const [navigation, setNavigation] = useState<Nullable<AppTabNavigation>>(null)
 
-  const context = useMemo(
+  const navigationContext = useMemo(
     () => ({ navigation, setNavigation }),
```

Comment:
should we remove `setNavigation` from this context?

Reply:
oh yeah 100% nice catch. it shouldn't technically matter, but prob good practice